packages/orchestration/src/examples/sendAnywhere.contract.js

```diff
@@ -30,6 +30,44 @@ const { Fail } = assert;
  * }} OrchestrationPowers
  */
 
+const guestFn = async (
```

Comment:
@turadg , here I used your suggestion to spot inappropriate lexical captures, by moving the guest function itself to the top of the file. Worked great! Revealed that `findInBank` and `contractAccount` were the problems. `findAccount` I merely made into an endowment. `contractAccount` was tougher because the issue of lexical capture of a read/write `let` variable rather than a value. As @dckc and I discussed, I turned this into a state record endowment. Both seem to work, AFAICT.

Reply:
"guest" is odd terminology in this context.

```suggestion
const sendItFn = async (
```

Also, hoisting it lost a bunch of type info. Please restore it.

Reply:
Rename done.

Not sure how to "restore" type info here. Instead, I manually typed `sendItFn` as best I could figure out. AFAICT, done, so resolving. But PTAL and if I need to revisit, please unresolve.

Reply:
> I manually typed sendItFn a best I could figure out.

that's what I had in mind. IOU a double-check